src/app/page.tsx

```diff
@@ -9,6 +9,10 @@ export default async function Home() {
     <div className="space-y-4">
       <Heading level={1}>Home</Heading>
 
+      <span>
+        {session?.user?.email} # {session?.user?.id}
```

Comment:
This is proof that we can now get actual, persisted user data out of the database. It's temporary.